src/CommonITILObject.php

```diff
@@ -624,6 +628,14 @@ public function showForm($ID, array $options = [])
         $predefined_fields = $this->setPredefinedFields($tt, $options, static::getDefaultValues());
         $this->initForm($this->fields['id'], $options);
 
+        $options['_canupdate'] = Session::haveRight('ticket', CREATE);
+        $item_commonitilobject = null;
+        if ($options['_canupdate']) {
+            //compute related item object (Ticket havee his own showForm)
+            $item_commonitilobject = get_called_class() === Change::class ? new Change_Item() : new Item_Problem();
```

Comment:
Use `getItemLinkClass` instead as plugins can add other classes that extend `CommonITILObject`. Releases plugin at least already does that.

Reply:
done